src/openms/source/QC/MQEvidenceExporter.cpp

```diff
@@ -0,0 +1,517 @@
+    file_ << "Modification" << "\t";
+    file_ << "Modified Sequence" << "\t";
+    file_ << "Mass" << "\t";
+    file_ << "Score" << "\t";
+    file_ << "Protein" << "\t";
+    file_ << "Protein group IDs" << "\t";
+    file_ << "Charge" << "\t";
+    file_ << "M/Z" << "\t";
+    file_ << "Retention Time" << "\t";
+    file_ << "Retention Length" << "\t";
+    file_ << "Intensity" << "\t";
+    file_ << "Resolution" << "\t";
+    file_ << "Potential contaminant" << "\t";
+    file_ << "Type" << "\t";
+    file_ << "Missed cleavages" << "\t";
+    file_ << "Mass error [ppm]" << "\t";
+    file_ << "Uncalibrated Mass error [ppm]" << "\t";
+    file_ << "Mass error [Da]" << "\t";
+    file_ << "Uncalibrated Mass error [Da]" << "\t";
+    file_ << "Uncalibrated - Calibrated m/z [ppm]" << "\t";
+    file_ << "Uncalibrated - Calibrated m/z [Da]" << "\t";
+    file_ << "Calibrated retention time start" << "\t";
+    file_ << "Calibrated retention time end" << "\t";
+    file_ << "Calibrated Retention Time" << "\t";
+    file_ << "Retention time calibration" << "\t";
+    file_ << "MS/MS count" << "\t";
+    file_ << "Match time difference" << "\t";
+    file_ << "Match m/z difference" << "\t";
+    file_ << "Raw file" << "\t";
+    file_ << "\n";
+}
+/*
+ * Description:
+ * Returns a distinct number for each protein
+ */
+UInt64 MQEvidence::proteinGroupID(const String &protein)
+{
+    auto it = protein_id_.find(protein);
+    if(it == protein_id_.end())
+    {
+        protein_id_.emplace(protein, protein_id_.size()+1);
+        return protein_id_.size();
+    }
+    else
+    {
+        return it -> second;
+    }
+}
+
+/*
+ * Description:
+ * Returns a pair that includes the index of the PeptideIdentification
+ * the best PeptideHit is refering to and a pointer to the best PeptideHit itself
+ *
+ */
+std::pair<Size , const PeptideHit *> MQEvidence::getBestPeptideHit(const std::vector<PeptideIdentification>& pep_ids)
+{   const PeptideHit * best = &pep_ids[0].getHits()[0];
+    Size pep_id = 0;
+    for(Size i = 0; i < pep_ids.size();++i)
+    {
+        for (const PeptideHit &hit : pep_ids[i].getHits()) {
+            if (best->getScore() < hit.getScore())
+            {
+                best = &hit;
+                pep_id = i;
+            }
+        }
+    }
+    return std::make_pair(pep_id,best);
+
+}
+
+/*
+ * Description:
+ * Returns a map that includes the information
+ * which feature is mapped to which consensus feature
+ */
+std::map<UInt64, Size> MQEvidence::mapFeatureIDtoConsensusID(const ConsensusMap & cmap)
+{
+    std::map<UInt64, Size> f_to_ci;
+    for(Size i = 0; i < cmap.size(); ++i)
+    {
+        for(const auto & fh : cmap[i].getFeatures())
+        {
+            f_to_ci[fh.getUniqueId()] = i;
+        }
+    }
+    return f_to_ci;
+}
+
+/*
+ * Description:
+ * Input is the path of a file.
+ * Function returns the filename without the information about the path
+ */
+String pathDeleter(const String & path)
```

Comment:
remove this function. use `File::basename()` (pull from OpenMS/develop first to obtain the latest version)